docs/fides/docs/ci_reference.md

```diff
@@ -0,0 +1,63 @@
+# CI Reference
+
+Fidesctl is a CI/CD tool first and foremost. With that in mind, we've included example CI/CD files for GitHub Actions to give an idea of what it looks like to run Fidesctl as part of a software development workflow.
+
+You should plan to implement two CI actions:
+
+1. Run `fidesctl evaluate --dry <resource_dir>` on pull requests, to check if new code will be accepted without applying those changes to the server.
+2. Run `fidesctl evaluate <resource_dir>` on commits to main, to synchronize the latest changes to the server.
+
+Let's see what those look like for GitHub Actions:
+
+```yaml
+name: Fidesctl CI Check
+
+# Only check on Pull Requests that target main
+on:
+  pull_request:
+    branches:
+      - main
+    paths: # Only run checks when the resource files change or the workflow file changes
+      - fides_resources/**
+      - .github/workflows/fidesctl_ci.yaml
+
+jobs:
+  Fidesctl:
+    runs-on: ubuntu-latest
+    container:
+      image: ethyca/fidesctl:latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Dry Evaluation
+        run: fidesctl evaluate --dry fides_resources/
+        env:
+          FIDESCTL__CLI__SERVER_URL: "fidesctl.privacyco.com"
```

Comment:
Haven't tried it, but do we need the protocol in front of the URL? We have it in other references.

```suggestion
          FIDESCTL__CLI__SERVER_URL: "https://fidesctl.privacyco.com"
```


If so, we should also change line 62.